src/router/index.js

```diff
@@ -29,6 +29,7 @@ const EmptyParentComponent = {
 }
 
 export default new Router({
+  mode: 'history',
```

Comment:
Not sure if we need to enable it by default.

@asvae , Maybe something that we might want to add to wiki?

Reply:
it helps to prevent "#" in url (i saw this solution on one of the forums), related issue:
https://github.com/epicmaxco/vuestic-admin/issues/285

Reply:
Doesn't seem related. In my understanding the issue is that if you click on a category and then refresh the page you get redirected to the main page. Am I correct?

Reply:
Let's say you are on `https://vuestic.epicmax.co/#/admin/forms/form-elements`. If you click on a category the url goes `https://vuestic.epicmax.co/#/` which is incorrect. Url should not change.

Reply:
oh, i understand. tnx!

Reply:
Mode history requires quite specific server config. And we want vuestic to be simple to start.